pkg/model/manifest.go

```diff
@@ -686,11 +686,11 @@ func (m *Manifest) SanitizeSvcNames() error {
 		}
 	}
 
-	for _, gf := range m.GlobalForward {
-		if shouldBeSanitized(gf.ServiceName) {
-			sanitizedSvcName := sanitizeName(gf.ServiceName)
-			gf.ServiceName = sanitizedSvcName
-			sanitizedServicesNames[gf.ServiceName] = sanitizedSvcName
+	for indx := range m.GlobalForward {
+		if shouldBeSanitized(m.GlobalForward[indx].ServiceName) {
```

Comment:
Could we extract `m.GlobalForward[indx].ServiceName` at the beginning of the for loop? I think it will look cleaner than having to read such a long variable name